features/20-standalone-launch.feature

```diff
@@ -31,6 +31,14 @@ Feature: Follows the SMART standalone launch sequence
         When I ask for authorization without the state field
         Then the response code should not be 200
 
+    Scenario: EHR evaluates authorization request - very long "state"
```

Comment:
I wouldn't think it's a high-yield test; does anybody fail it? Doesn't hurt anyway, as a scaffolding!